lib/python/src/bailo/core/agent.py

```diff
@@ -106,18 +115,27 @@ def __init__(
         :param secret_key: Secret key
         """
         super().__init__()
+        logger.info("TokenAgent initiated successfully.")
 
         if access_key is None:
+            logger.error("Access key not provided. Trying other sources...")
             try:
                 access_key = os.environ["BAILO_ACCESS_KEY"]
+                logger.info("Access key acquired from BAILO_ACCESS_KEY environment variable.")
             except KeyError:
+                logger.error("Access key not found in BAILO_ACCESS_KEY environment variable. Requires user input.")
```

Comment:
Normally logging errors are given when something is unrecoverable within the application. I think info should be used here instead

Reply:
Done